dask_geomodeling/core/graphs.py

```diff
@@ -23,7 +23,7 @@ def _construct_exc_callback(e, dumps):
 
     """
     key = inspect.currentframe().f_back.f_locals.get("key")
-    e.args = (f"{key}: {str(e)}",)
+    e.args = ("{0}: {1}".format(key, str(e)),)
     raise
```

Comment:
Is een losse `raise` hier genoeg? Moet het niet `raise e` zijn?

Reply:
https://docs.python.org/3/reference/simple_stmts.html#index-27

"If no expressions are present, raise re-raises the last exception that was active in the current scope. If no exception is active in the current scope, a RuntimeError exception is raised indicating that this is an error."

Maar *explicit is better than implicit* dus ik verander hem